src/components/personal-information/index.js

```diff
@@ -39,6 +39,26 @@ const PersonalInfoComponent = ({
 
     const initialFirstName = userProfile.given_name || (invitation ? invitation.first_name : '');
     const initialLastName = userProfile.family_name || (invitation ? invitation.last_name : '');
+    
+    const [ticketOwnerOption, setTicketOwnerOption] = useState('');
+    const [ticketOwnerError, setTicketOwnerError] = useState(false);
+    
+    const radioListOptions = [
+        {label: "Myself", value: 'myself'},
+        {label: "Someone Else", value: 'someoneElse'},
+        {label: "Leave Unassigned", value: 'unassigned'},
+    ]
+
+    const handleRadioButtonChange = (ev) => {
+        const {value} = ev.target;
+        const selfTicket = value === 'myself';        
+        setTicketOwnerOption(value);
+        setTicketOwnerError(false);
+        setPersonalInfo({ ...personalInfo, ticketForMyself: selfTicket, attendee: { firstName: '', lastName: '', email: '' }});
```

Comment:
dont do this assign attendee data using the options 
myself -> attendee data is purchaser info
someElse -> user provided data